source/_current_clients.html.erb

```diff
@@ -3,10 +3,11 @@
   <ul class="collapsible popout" data-collapsible="accordion">
     <li>
       <div class="collapsible-header active"><i class="material-icons">play_arrow</i>
-        Lead with Cornerstone
+        Cornerstone Management Resource Systems
       </div>
       <div class="collapsible-body">
-        <h5>Lead with Cornerstone ReadyDayOne</h5>
+        <h5><a href="leadwithcornerstone.com">Cornerstone Management Resource Systems</a></h5>
```

Comment:
when calling out to a new domain we have to include the `protocol://` otherwise the browser will attach it to the current domain